MTFO/Utilities/Json.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.IO;
+using System.Text.Json;
+
+namespace MTFO.Utilities
+{
+    public static class Json
+    {
+        /// <summary>
+        /// Checks if a file exists and will deserialize it using the given type, otherwise it will construct the output and write it to a file
+        /// </summary>
+        public static void TryRead<T>(string path, out T output) where T : new()
```

Comment:
With this utility class being public, an option to edit the `JsonSerializerOptions` should be provided for more control over the (de)serialization.

The old ContentManager implementation for JSON didn't provide that but if we're moving away from it it's best we provide extra control.

Reply:
Made the class non-static so it can be constructed with preferred values, I would prefer keeping JsonSerializerOptions internalized and eventually implementing parameters that can modify the settings.